### DISCUSSION TITLE
Is it possible to access jzz instance from webmidi?

### DISCUSSION THREAD

**Discussion by maximelebreton** [djipco/webmidi#406] [General]
Hello, and thanks for your amazing work on webmidi!

I'm using a plugin named `jzz-midi-gear`, based on `jzz`,
but I'm also using `webmidi`, so I would like to remove my `jzz` depedency, and using the `jzz` instance from `webmidi`.

Is that possible?
I hope i'm clear :)

Thanks!

**Comment by djipco** [comment]
The `jzz` module is imported directly into `global.navigator`. Therefore, you can use `global.navigator` as your `jzz` instance. This is how the Jazz-Soft people do it internally.

**Comment by maximelebreton** [comment]
Thanks @djipco!
